libs/rs-sdk-integration-tests/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "rs-sdk-integration-tests"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+seda-sdk-rs = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+hex = "0.4"
```

Comment:
Let's preferably do this dep the same as above.

For shorthand you can also do `dep_name.workspace = true`